src/Sprites/spriteManager.ts

```diff
@@ -401,6 +410,68 @@ export class SpriteManager implements ISpriteManager {
         return null;
     }
 
+    /**
+     * Intersects the sprites with a ray
+     * @param ray defines the ray to intersect with
+     * @param camera defines the current active camera
+     * @param predicate defines a predicate used to select candidate sprites
+     * @returns null if no hit or a PickingInfo array
+     */
+    public multiIntersects(ray: Ray, camera: Camera, predicate?: (sprite: Sprite) => boolean): Nullable<PickingInfo[]> {
+        var count = Math.min(this._capacity, this.sprites.length);
+        var min = Vector3.Zero();
+        var max = Vector3.Zero();
+        var distance: number;
+        var results: Nullable<PickingInfo[]> = [];
+        var pickedPoint = Vector3.Zero();
+        var cameraSpacePosition = Vector3.Zero();

```

Comment:
and here :)

Reply:
done

Reply:
Forgot cameraSpacePosition - comes of trying to do changes in a hurry